Lib/test/test_urlparse.py

```diff
@@ -140,7 +140,7 @@ def test_qsl(self):
             expect_without_blanks = [v for v in expect if len(v[1])]
             result = urllib.parse.parse_qsl(orig, keep_blank_values=False)
             self.assertEqual(result, expect_without_blanks,
-                            "Error parsing %r" % orig)
+                             "Error parsing %r" % orig)
```

Comment:
There are simple format changes not related to the unsupported test marking. Will you please remvoe these changes?